apps/src/code-studio/components/libraries/LibraryCreationDialog.jsx

```diff
@@ -123,6 +137,8 @@ class LibraryCreationDialog extends React.Component {
           <PublishSuccessDisplay
             libraryName={libraryName}
             channelId={channelId}
+            // Waiting for design guidance prior to enabling this functionality
+            // onShareTeacherLibrary={this.onShareTeacherLibrary()}
```

Comment:
i know this is commented out, but just a heads up that the parentheses will need to be removed here:
```suggestion
            // onShareTeacherLibrary={this.onShareTeacherLibrary}
```